includes/class-dsar-shortcode.php

```
@@ -0,0 +1,406 @@
+			$admin_email,
+			/* translators: 1: site name, 2: request type */
+			sprintf( __( '[%1$s] Data Subject Request: %2$s', 'faz-cookie-manager' ), $site_name, $type_label ),
+			$body,
+			array( 'Reply-To: ' . $safe_name . ' <' . $email . '>' )
+		);
+	}
+
+	/**
+	 * Send a confirmation email to the requester.
+	 */
+	private function send_confirmation( $name, $email, $type ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$type'. (undefined)

(UnusedFormalParameter)